wiki_articles/vscode_cpp_config.md

```diff
@@ -0,0 +1,26 @@
+# How to configure VSCode for C and C++
+This guide assumes that you have installed clangd, clang-format, a C compiler 
+and they are in PATH. If not, please check our article on how to do so with msys2 and gcc.
+
+## Language Server
+Microsoft has their own intellisense extension for C and C++. However, from the personal
+experience of the author of this article, `clangd` is just much better.
+1. Open up Marketplace and install `clangd` by LLVM
+2. It should automatically detect clangd and should work as-is.
+
+## Code Formatter
+1. Install `Clang-Format` Extension by Xaver Hellauer from Marketplace.
+2. Press `Ctrl+,` keys, it should open up VSCode settings.
+3. Search for `formatter`, and then set the `Default Formatter` setting to `clang-format`.
```

Comment:
I think I'd recommend omitting the formatter setup. I'm open to leaving it if you feel strongly however I think it isn't as pertinent to someone who needs this guide to get vscode setup.

Reply:
I don't feel that strongly about it. I'll get rid of it.